src/utils/platform.ts

```diff
@@ -13,3 +13,13 @@ export const getPlatformShortName = (
   if (platformName.toLowerCase().includes('android')) return 'Android';
   return 'Unspecified';
 };
+
+export const getPlatformAndVersion = (platform: string): string => {
+  const platformShortName = getPlatformShortName(platform);
+  // Example of android platform, the split is specific for this platform
+  // Other platforms are built differently
+  // android-hw-p6-13-0-android-aarch64-shippable-qr
+  if (platformShortName === 'Android')
```

Comment:
I feel like we need to check if platform with `android-hw-` to know if that's the right format. Otherwise we break when the format isn't what we expect (for example select "build metrics" with your deploy preview link).

This will avoid the issues with our tests too.

```suggestion
  if (platform.startsWith('android-hw-'))
```

Reply:
In the tests there is an example `'android-em-7-0-x86_64-lite-qr'`.
The check for `'android-hw-'` breaks for this.

Reply:
-em- is for emulator so I believe it's irrelevant to your change, so on the contrary it does what we want :-)